assets/javascripts/tests.js

```diff
@@ -184,26 +184,37 @@ function changeJobPrio(jobId, delta, linkElement) {
     });
 }
 
+function renderTestSummary(data) {
+    var html = ''
+
+    if (data.passed)
+        html += data.passed + "<i class='fa module_passed fa-star' title='modules passed'></i>";
+    else
+        html += "0 <i class='fa module_passed fa-star' title='modules passed'></i>";
```

Comment:
This is a little bit repetitive. Why not just `html += (data.passed || 0) + "…`? And I also don't think all these blank lines after each `if` block make it easier to read.

Reply:
yes, good optimization